### PR TITLE
[Bugfix] Type of inputs are always string fix

### PR DESCRIPTION
**Pull request by ViacheslavKudinov**
- fixed that `type` input hardcoded as `string`

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -145,6 +145,7 @@ jobs:
             README.md
             test/**
 
+
```
Comment (jackton1) [inline .github/workflows/test.yml:148]: Hi @ViacheslavKudinov can we remove the extra space
```suggestion
```

File: internal/types/reusable.go [hunk 2]
```diff
@@ -27,17 +27,17 @@ import (
 
 	"github.com/olekukonko/tablewriter"
 	"github.com/spf13/cobra"
-	"gopkg.in/yaml.v3"
-
 	"github.com/tj-actions/auto-doc/internal"
 	"github.com/tj-actions/auto-doc/internal/utils"
+	"gopkg.in/yaml.v3"
```
Comment (jackton1) [inline internal/types/reusable.go:32]: Since this a third-party package we can revert this change

**Review by jackton1 — CHANGES_REQUESTED** [review]

**Comment by ViacheslavKudinov** [comment]
@jackton1 Could you please take a look?`
PS Not sure how `pull_request_target` has to work there https://github.com/tj-actions/auto-doc/blob/main/.github/workflows/test.yml#L4

File: internal/types/reusable.go (hunk 2 above)
Comment (ViacheslavKudinov) [reply internal/types/reusable.go:32]: Sure, reverted.

Comment (jackton1) [reply internal/types/reusable.go:32]: Actually this appears to be automated

Comment (ViacheslavKudinov) [reply internal/types/reusable.go:32]: Yes, exactly.

**Review by jackton1 — APPROVED** [review]
